### PR TITLE
Mention default branch in developer documentation

### PR DESCRIPTION
**Pull request by cachedout**
Closes https://github.com/elastic/beats/issues/9532

### CODE REVIEW

File: docs/devguide/contributing.asciidoc [hunk 1]
```diff
@@ -25,7 +25,9 @@ You only need to sign the CLA once.
 
 . Send a pull request! Push your changes to your fork of the repository and
 https://help.github.com/articles/using-pull-requests[submit a pull request] using our
-<<pr-review,pull request guidelines>>. In the pull request, describe what your changes do and mention
+<<pr-review,pull request guidelines>>. Most new PRs will go to the master branch.
```
Comment (kvch) [inline docs/devguide/contributing.asciidoc]: Most? I thought all PRs should go to master and then get backported the required versions.

Comment (cachedout) [reply docs/devguide/contributing.asciidoc]: I just softened the language to guard against a case where there might be a special exception, though I don't know offhand specifically what that might be. I'm fine with just removing the word `Most`.

Comment (kvch) [reply docs/devguide/contributing.asciidoc]: I would mention backporting fixes from master. I have seen a few community PRs where the author opened the fix to the specific version he/she found the issue in.

Comment (cachedout) [reply docs/devguide/contributing.asciidoc]: Should community members submit backports after their PR to master is accepted? Or should they assume that the Beats team will handle this for them?

Comment (kvch) [reply docs/devguide/contributing.asciidoc]: we handle the backporting usually.

Comment (cachedout) [reply docs/devguide/contributing.asciidoc]: Added sentence about backporting.

**Pull request merged**